frontend/src/metabase/query_builder/components/expressions/ExpressionEditorSuggestions/ExpressionEditorSuggestions.tsx

```diff
@@ -109,60 +115,62 @@ export function ExpressionEditorSuggestions({
       <Popover.Target>{children}</Popover.Target>
       <Popover.Dropdown>
         <DelayGroup>
-          <ExpressionList
-            data-testid="expression-suggestions-list"
-            ref={ref}
-            onMouseDownCapture={handleMouseDown}
-          >
-            <ExpressionEditorSuggestionsListGroup
-              suggestions={groups._none}
-              query={query}
-              stageIndex={stageIndex}
-              highlightedIndex={highlightedIndex}
-              onSuggestionMouseDown={onSuggestionMouseDown}
-              onHighlightSuggestion={onHighlightSuggestion}
-            />
-            <ExpressionEditorSuggestionsListGroup
-              name="popularAggregations"
-              suggestions={groups.popularAggregations}
-              query={query}
-              stageIndex={stageIndex}
-              highlightedIndex={highlightedIndex}
-              onSuggestionMouseDown={onSuggestionMouseDown}
-              onHighlightSuggestion={onHighlightSuggestion}
-            />
-            <ExpressionEditorSuggestionsListGroup
-              name="popularExpressions"
-              suggestions={groups.popularExpressions}
-              query={query}
-              stageIndex={stageIndex}
-              highlightedIndex={highlightedIndex}
-              onSuggestionMouseDown={onSuggestionMouseDown}
-              onHighlightSuggestion={onHighlightSuggestion}
-            />
-            <ExpressionEditorSuggestionsListGroup
-              name="shortcuts"
-              suggestions={groups.shortcuts}
-              query={query}
-              stageIndex={stageIndex}
-              highlightedIndex={highlightedIndex}
-              onSuggestionMouseDown={onSuggestionMouseDown}
-              onHighlightSuggestion={onHighlightSuggestion}
-            />
-          </ExpressionList>
-          {footers.map(suggestion => (
-            <Footer
-              key={suggestion.index}
-              suggestion={suggestion}
-              highlightedIndex={highlightedIndex}
-              onHighlightSuggestion={onHighlightSuggestion}
-            />
-          ))}
+          <div ref={ref}>
```

Comment:
Do we need to attach `ref` to 2 different elements here or could we use `useMergedRef` to avoid adding an extra div?

Reply:
@kamilmielnik I was hesitant to do it since `listRef` was used internally, but it does make sense. Let me update it